data/strings/types_strings.txt

```diff
@@ -26936,7 +26939,7 @@
 
   [type.piste_type.downhill]
     en = Downhill Ski Run
-    de = Abfahrt
+    de = Ski Abfahrt
```

Comment:
Ski-Abfahrt

Reply:
To make it easier to apply the suggestion for @map-per ([see here](https://haacked.com/archive/2019/06/03/suggested-changes)):

```suggestion
    de = Ski-Abfahrt
```